src/DataMigration/custom/Cmdlets/Get-AzDataMigrationAssessment.ps1

```diff
@@ -0,0 +1,132 @@
+
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+<#
+.Synopsis
+Run assessment on given Sql Servers.
+.Description
+Run assessment on given Sql Servers.
+#>
+
+
+function Get-AzDataMigrationAssessment 
+{
+    [OutputType([System.Boolean])]
+    [CmdletBinding(PositionalBinding=$false)]
+    [Microsoft.Azure.PowerShell.Cmdlets.DataMigration.Description('Start assessment on SQL Server instance(s)')]
+
+    param(
+        [Parameter(ParameterSetName='CommandLine', Mandatory, HelpMessage='Sql Server Connection Strings')]
+        [System.String[]]
+        ${ConnectionString},
+
+
+        [Parameter(ParameterSetName='CommandLine', HelpMessage='Output folder to store assessment report')]
+        [System.String]
+        ${OutputFolder},
+
+
+        [Parameter(ParameterSetName='CommandLine', HelpMessage='Enable this parameter to overwrite the existing assessment report')]
+        [System.Management.Automation.SwitchParameter]
+        ${Overwrite},
+
+        [Parameter(ParameterSetName='ConfigFile', Mandatory, HelpMessage='Path of the ConfigFile')]
+        [System.String]
+        ${ConfigFilePath},
+
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.DataMigration.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Returns true when the command succeeds
+        ${PassThru}
+    )
+
+    process 
+    {
+        try 
+        {
+
+            #Defining Default Output Path
+            $DefaultOutputFolder = Get-DefaultOutputFolder
+
+            #Defining Base and Exe paths
+            $BaseFolder = Join-Path -Path $DefaultOutputFolder -ChildPath Downloads;
+            $ExePath = Join-Path -Path $BaseFolder -ChildPath SqlAssessment.Console.csproj\SqlAssessment.exe;
```

Comment:
Do you also need a check for os type here?

Reply:
Yes, we are doing that in helper functions. This is where we are doing it: https://github.com/ashutoshsuman99/azure-powershell/blob/AddingDataMigrationCmdlet/src/DataMigration/custom/Helpers/Assessment.ps1#L33. We return OS dependent paths to $DefaultOutputFolder, which we later use in our base folder.

Reply:
> Yes, we are doing that in helper functions. This is where we are doing it: https://github.com/ashutoshsuman99/azure-powershell/blob/AddingDataMigrationCmdlet/src/DataMigration/custom/Helpers/Assessment.ps1#L41. We return OS dependent paths to $DefaultOutputFolder, which we later use in our base folder.

And so the SqlAssessment on Linux and Mac are also called "SqlAssessment.exe“?

Reply:
Just verified with the SqlAssessment team, currently Linux and  Mac support for this is not available. So even in our command we have added a restriction for this command when running in this OS environments.